protos/dcs/coalition/v0/coalition.proto

```diff
@@ -131,6 +135,16 @@ message AddGroupResponse {
   dcs.common.v0.Group group = 1;
 }
 
+message GetStaticObjectsRequest {
+  // the coalition which the statics belong to
+  dcs.common.v0.Coalition coalition = 1;
+}
+
+message GetStaticObjectsResponse {
+  // the list of statics
+  repeated dcs.common.v0.Static statics = 1;
```

Comment:
Nit: I _think_ that, so far, the return field has matched the request. So `groups` for `getGroups` etc. So maybe this would be better named `static_objects`.

Not too fussed about it though. Hence just a nit.